libs/core/web/src/lib/pages/admin/Admin.tsx

```diff
@@ -1,11 +1,78 @@
-/* eslint-disable-next-line */
-export interface AdminProps {}
+import { Box } from '@mui/material';
+import LoadingButton from '@mui/lab/LoadingButton';
+import Typography from '@mui/material/Typography';
+import axios from 'axios';
+import { useEffect, useState } from 'react';
```

Comment:
is typescript not throwing an error for this file when there's no React import ?
```tsx
import React, { useEffect, useState } from 'react';
```